css/style.css

```diff
@@ -502,3 +528,349 @@ form {
 .form-group label {
   display: none;
 }
+
+hr {
+  display: none;
+}
+
+.first-last-name {
+  display: none;
+}
+
+.contact-illus-1,
+.contact-illus-2 {
+  display: none;
+}
+
+/* --------media queries----- */
+@media (min-width: 768px) {
+  body {
+    background-image: url(../images/Header-llustration-desktop.svg);
```

Comment:
## OPTIONAL

- [ ] Kindly add this image to the headline section instead of the body, please see Figma design and also add the appropriate spacing before my recent works section 🤝 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![ww](https://user-images.githubusercontent.com/69638013/160089016-7f50503b-b3bd-4341-b7b5-90a1183c48dc.png)  |  ![Screenshot_38](https://user-images.githubusercontent.com/69638013/160089056-f8429e22-2ae3-4223-91c5-df7146342729.png)

- Doing this will make your design look like the Figma design

Reply:
resolved 👍